I'm sorry, but an uncaught exception occurred.

While loading <'Image' u'Cen\xe1rios/Sala n\xfamero 1/bg sala1 itens vazio.png'>:
  File "game/script.rpy", line 408, in script
    call screen escritorio
  File "renpy/common/000statements.rpy", line 569, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
IOError: Couldn't find file 'Cenários/Sala número 1/bg sala1 itens vazio.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 408, in script
    call screen escritorio
  File "/usr/share/renpy/renpy/ast.py", line 2015, in execute
    self.call("execute")
  File "/usr/share/renpy/renpy/ast.py", line 2003, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/usr/share/renpy/renpy/statements.py", line 278, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 569, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "/usr/share/renpy/renpy/exports.py", line 3136, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "/usr/share/renpy/renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/usr/share/renpy/renpy/display/core.py", line 3325, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "/usr/share/renpy/renpy/display/core.py", line 3839, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "/usr/share/renpy/renpy/display/core.py", line 2565, in draw_screen
    renpy.config.screen_height,
  File "render.pyx", line 492, in renpy.display.render.render_screen
  File "render.pyx", line 263, in renpy.display.render.render
  File "/usr/share/renpy/renpy/display/layout.py", line 826, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/usr/share/renpy/renpy/display/layout.py", line 826, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/usr/share/renpy/renpy/display/layout.py", line 826, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/usr/share/renpy/renpy/display/screen.py", line 688, in render
    child = renpy.display.render.render(self.child, w, h, st, at)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/usr/share/renpy/renpy/display/layout.py", line 826, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/usr/share/renpy/renpy/display/im.py", line 625, in render
    return cache.get(self, render=True)
  File "/usr/share/renpy/renpy/display/im.py", line 287, in get
    surf = image.load()
  File "/usr/share/renpy/renpy/display/im.py", line 706, in load
    raise e
IOError: Couldn't find file 'Cenários/Sala número 1/bg sala1 itens vazio.png'.

Linux-5.18.9-zen1-1-zen-x86_64-with-glibc2.2.5
Ren'Py 7.4.11.2266u
Mansao Macambira 1.0
Tue Jul 19 01:07:51 2022
